webpack/assets/javascripts/react_app/components/common/forms/FormField.fixtures.js

```diff
@@ -15,6 +15,20 @@ export const dateTimeWithErrorProps = {
   error: 'can not be in the past',
 };
 
+
+export const selectProps = {
+  type: 'select',
+  id: 'date-time',
+  label: 'Grouped select',
+  name: 'group[groupedSelect]',
+  options: [
+    { true: 'Ungrouped true' },
+    { groupLabel: 'Group1', children: { rnd: 'Group1 opt1' }},
+    { false: 'Ungrouped false' },
+    { groupLabel: 'Group2', children: [['rnd2', 'Group2 opt2']]},
```

Comment:
Should the label in the Group2 child be 'Group2 opt2'?

![](https://i.imgur.com/jnoa7p3.png)